ovmf-vars-generator

```diff
@@ -136,9 +140,10 @@ def test_keys(args):
             read = p.stdout.readline()
             if args.print_output:
                 print('OUT: %s' % strip_special(read), end='')
-            if 'Secure boot disabled' in read:
+                print()
+            if b'Secure boot disabled' in read:
```

Comment:
Ultra nit: s/Secure boot/Secure Boot/g/

Reply:
This is the output from the Linux kernel, so we can't really change this.

Reply:
Hehe, indeed.  I did think of it for a moment, and then realized it after hitting submit on the comment. Disregard it, then.